File: siserp-logistica-backend/src/main/java/com/besoft/siserp/lgtc/dto/request/ProductoCargarExcelRequest.java
package com.besoft.siserp.lgtc.dto.request;

import java.io.Serializable;
import java.util.Date;
import java.util.List;

import lombok.Getter;
import lombok.ToString;
import lombok.Setter;

@Getter
@Setter
@ToString
public class ProductoCargarExcelRequest implements Serializable {
	private static final long serialVersionUID = 1L;

	private Long idtTipo;
	private Long idEstante;
	private Long idUsuarioCrea;
	private Date fecUsuarioCrea;
	private List<ProductoCargarExcelDetRequest> listaProducto;
}

File: siserp-logistica-frontend/src/main/siserp-logistica-frontend/src/app/modules/intranet/dto/request/producto-cargar-excel.request.ts
import { ProductoCargarExcelDetRequest } from "./producto-cargar-excel-det.request";

export class ProductoCargarExcelRequest {
    idtTipo: number;
    idEstante: number;
    idUsuarioCrea: number;
    fecUsuarioCrea: Date;
    listaProducto: ProductoCargarExcelDetRequest[];
}